infra/modules/providers/azure/app-service/variables.tf

```diff
@@ -20,12 +20,24 @@ variable "resource_tags" {
   default     = {}
 }
 
-variable "app_service_name" {
+variable "app_service_config" {
   description = "The name of the app service to be created"
-  type        = map(string)
+  type        = map(map(string))
   default     = {}
 }
 
+variable "enable_auth" {
```

Comment:
See note about removing this in favor of defaulting to the app-specific config.

Reply:
app-specific config implemented. Enable_auth removed.